splinterd/src/node/builder/mod.rs

```diff
@@ -58,6 +60,7 @@ pub struct NodeBuilder {
     network_subsystem_builder: NetworkSubsystemBuilder,
     node_id: Option<String>,
     auth_configs: Vec<AuthConfig>,
+    biome_actix1_resources: Vec<Actix1Resource>,
```

Comment:
These are tied to the RestApIVariant, so they shouldn't necessarily be top-level items.  Like with other resources, they may need to be moved to later steps, then the build step.

Reply:
I altered this so that the biome resources are being constructed at the `RunnableNode` stage (since the resources need access to the store factory) so I was able to take these resources out from the top-level and replaced this with just a `biome_enabled` flag for the `NodeBuilder`